app/states/texas/texas_search.py

```
@@ -1,173 +1,294 @@
+        self.config = TexasSearchSetup(self.data)
+        self.data = self.config.DATA
 
-@dataclass
-class TECSearchQuery:
-    data: TECCriteriaPrompts = field(default_factory=TECCriteriaPrompts)
-    __session: SessionLocal = SessionLocal
+    def normal_search(self, result: TexasSearchResult) -> tuple[list[DataFrame], list[DataFrame], dict[
+        Any, dict[int, Any]]] | None:
+        _result = []
+        _unique_filers = []
+        _unique_filters = dict()
```

Ruff (0.14.7)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)